tests/test_subtractTask.py

```diff
@@ -724,6 +724,278 @@ def _compare_apCorrMaps(self, a, b):
                 value.getCoefficients(), value2.getCoefficients(), rtol=0.0)
 
 
+class AlardLuptonPreconvolveSubtractTest(lsst.utils.tests.TestCase):
+
+    def test_mismatched_template(self):
```

Comment:
This implies to me that there could be a mixin class to contain these tests that are shared; at least this one,  `test_few_sources`, `test_equal_images`, `test_background_subtraction `, maybe others?